plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -393,15 +393,14 @@ class ServicesTable extends React.Component {
   }
 
   renderInstances(prop, service) {
+    const isFolder = service.isFolder();
```

Comment:
Please use the service type to check wether a service a _folder_ / tree of services or a plain service.